change/@ni-nimble-components-5ea84983-4985-4423-9b3c-0cec94b65700.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Ironic that the PR to fix intermittent FF tests has a failing build due to an intermittent webkit test 🤦